Support .nvmrc (#338)
actions/setup-node: docs/advanced-usage.md

- run: npm test
```

## Node version file

> The `node-version-file` input allows you to use a file within your repository which contains the version of node your project uses for example `.nvmrc`. If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used.

Reviewer: I think it makes sense to add a link to the [Supported version syntax](https://github.com/actions/setup-node#supported-version-syntax) section

Reviewer 2: @MaksimZhukov im wondering if we should remove  `If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used.` since it will throw a warning to the user for using both inputs

> > The node version file is read from the project root

Reviewer: Could you please clarify why this comment is resolved? Do you have any objections related to the proposed wording?

> > The node version file is read from the project root

Author: Actually i changed the line with what you suggested

> > The node version file is read from the project root

Author: ![image](https://user-images.githubusercontent.com/56452820/139793603-78622cc1-c749-4c1d-947f-a614ea334eb5.png)

> > The node version file is read from the project root

Author: changes

> > The node version file is read from the project root

Reviewer: @hkaur008, I see, thank you!

Sorry for the unclear comment. I expected you to replace the following line with the suggested one:
```
The node version file is read from the project root
```
-->
```
The action will search for the node version file relative to repository root
```

> > The node version file is read from the project root

Author: oh i see , done !

- run: npm test
```

## Node version file
  

> The `node-version-file` input contains the version of node used by project, for example `.nvmrc`. If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used. 

Reviewer: ```suggestion
The `node-version-file` input accepts path to file containing the version of Node.js used by project, for example `.nvmrc`. If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used. 
```

> The `node-version-file` input accepts path to file containing the version of Node.js used by project, for example `.nvmrc`. If both the `node-version` and the `node-version-file` inputs are provided the `node-version` input is used. 

Reviewer 3: ```suggestion
The `node-version-file` input accepts a path to a file containing the version of Node.js used by a project, for example `.nvmrc`. If both the `node-version` and the `node-version-file` inputs are provided then the `node-version` input is used. 
```

You can check [supported version syntax](https://github.com/actions/setup-node#supported-version-syntax)  

> > The action will search for the node version file relative to repository root.

Reviewer 3: ```suggestion
> The action will search for the node version file relative to the repository root.
```
